bin/build-locales

```diff
@@ -50,7 +51,17 @@ poFiles.forEach((pofile) => {
     var defineLocale = null;
     try {
       const momentLocale = locale.replace('_', '-').toLowerCase();
-      const localeModulePath = `moment/locale/${momentLocale}`;
```

Comment:
If you replace this with `getLocaleModulePath()` then you could add unit tests for it. I think it would be helpful to have tests for such a function.

Reply:
yep, good call. let's start testing those scripts.